packages/mdc-layout-grid/_variables.scss

```diff
@@ -36,6 +36,12 @@ $mdc-layout-grid-default-gutter: (
   phone: 16px
 ) !default;
 
+$mdc-layout-grid-column-width: (
```

Comment:
Maybe `$mdc-layout-grid-platform-column-widths`? It more closely follows the naming conventions of this pattern in other components.

It would obviously have to be changed in `_mixins.scss` as well